rlkit/envs/point_robot.py

```diff
@@ -8,13 +8,17 @@
 @register_env('point-robot')
 class PointEnv(Env):
     """
-    point robot on a 2-D plane
-    tasks (equivalent to goals here) sampled from the unit square
+    point robot on a 2-D plane with position control
+    tasks (aka goals) are positions on the plane
+
+     - tasks sampled from unit square
+     - reward is L2 distance
     """
 
     def __init__(self, randomize_tasks=False, n_tasks=2):
 
         if randomize_tasks:
+            np.random.seed(1337)
```

Comment:
but why

Reply:
my old collaborator used it for everything and I picked up the habit haha XD